packages/docs-reanimated/docs/core/makeMutable.mdx

```diff
@@ -0,0 +1,153 @@
+
+## Example
+
+import MakeMutable from '@site/src/examples/MakeMutable';
+import MakeMutableSrc from '!!raw-loader!@site/src/examples/MakeMutable';
+
+<InteractiveExample src={MakeMutableSrc} component={MakeMutable} showCode />
+
+## Remarks
+
+- All remarks from the [useSharedValue](/docs/core/useSharedValue) hook apply to `makeMutable` as well.
+
+- Don't use `makeMutable` directly in the component scope. When component re-renders, it will create the completely new object (with the new `initial` value if it was changed) and the state of the previous shared value will be lost.
+
+<Indent>
+
+```javascript
+function App() {
+  const [counter, setCounter] = useState(0);
+  const mv = makeMutable(counter); // creates a new shared value on each render
+
+  useEffect(() => {
+    const interval = setInterval(() => {
+      setCounter((prev) => prev + 1); // updates the counter stored in the component state
+    }, 1000);
+
+    return () => {
+      clearInterval(interval);
+    };
+  }, [mv]);
+
+  useAnimatedReaction(
+    () => mv.value,
+    (value) => {
+      console.log(value); // prints 0, 1, 2, ...
+    }
+  );
+}
+```
+
+</Indent>
+
+- Use `cancelAnimation` to stop all animations running on the shared value if it's no longer needed and there are still some animations running. Be super careful with infinite animations, as they will never stop unless you cancel them manually.
+
+<Indent>
+
+```javascript
+function App() {
+  const mv = useMemo(() => makeMutable(0), []);
+
+  useEffect(() => {
+    mv.value = withRepeat(withSpring(100), -1, true); // this creates an infinite animation
+
+    return () => {
+      cancelAnimation(mv); // this will stop the infinite animation when the component is unmounted
+    };
+  }, []);
+}
+```
+
+</Indent>
+
+- You don't have to use `cancelAnimation` when the value is not animated. It will be garbage collected automatically when no more references to it exist.
+
+<Indent>
+
+```javascript
+const someFlag = makeMutable(false);
+
+function App() {
+  someFlag.value = true; // no need to cancel the animation
+}
+```
+
+</Indent>
+
+### Comparison with `useSharedValue`
+
+| `makeMutable`                                                                       | `useSharedValue`                                                                                           |
+| ----------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------- |
+| Creates a new shared value object on each call                                      | Reuses the same shared value object on each call                                                           |
+| If `initial` value changes, a new shared value object with the new value is created | If `initialValue` value changes, the initially created shared value object is returned without any changes |
```

Comment:
```suggestion
| If `initial` value changes, a new shared value object with the new value is created | If `initial` value changes, the initially created shared value object is returned without any changes |
```

Reply:
I intentionally kept different names because different names are used in function definitions. 

The `useSharedValue` hook uses `initialValue` param name:

```tsx
export function useSharedValue<Value>(initialValue: Value): SharedValue<Value> {
  ...
}
```

whilst, the `makeMutable` uses the `initial` name of its parameter:

```tsx

export function makeMutable<Value>(initial: Value): Mutable<Value> {
  ...
}
```